src/niamoto/core/plugins/transformers/class_objects/categories_extractor.py

```
@@ -45,37 +34,28 @@ class ClassObjectCategoriesExtractor(TransformerPlugin):
+                details={"pydantic_error": e.errors(), "config": config},
             )
-
-        if not axis_config.get("output"):
+        except Exception as e:  # Catch other potential errors during validation
+            # Get plugin name from input config dict
+            plugin_name = config.get("plugin", "unknown_plugin")
             raise DataTransformError(
-                "axis.output must be specified", details={"config": config}
+                f"Unexpected error validating configuration for {plugin_name}",
+                details={"error": str(e), "config": config},
             )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)